pmlc/dialect/pxa/transforms/registration.cc

```diff
@@ -31,8 +31,9 @@ static mlir::PassRegistration<TestStrideInfoPass>
     regTestStrideInfo("test-stride-info",
                       "Report stride data for all loads/stores for unit tests");
 
-static mlir::PassRegistration<StencilPass>
+static mlir::PassPipelineRegistration<StencilPassOptions>
```

Comment:
I think we can register a pass without having to register a pipeline...

Reply:
It could be. I just followed the docs on Instance Pass Options at https://github.com/llvm/llvm-project/blob/master/mlir/docs/WritingAPass.md#instance-specific-pass-options
That is apparently how they do it.

Reply:
Unfortunately the docs to mention how to register a pass with instance-specific options, but it is a distinct thing from a pipeline with instance-specific options.

See the `TestPassManager.cpp` in the MLIR codebase

Reply:
Using that now.